src/main/java/testng-1.0.dtd.html

```diff
@@ -159,6 +159,9 @@
 Defines additional groups ("groups of groups") and also which groups to include in this test run
 --&gt;</span>
 &lt;!ELEMENT groups (define*,run?) &gt;
+<!ATTLIST groups
+    override-included-methods (true | false) "false"
```

Comment:
I like the idea to put the attribute in the schema but it means a new version of the schema should be released and supported by 3rd party tools.

I'd prefer a global configuration that will override the default behavior instead.

@krmahadevan thoughts?

Reply:
@juherr am with you on this. 

A JVM configuration and/or a TestNG configuration via command line would be less invasive to change

Reply:
@juherr @krmahadevan do you mean adding an option such as "-overrideincludedmethods" as shown here?
https://testng.org/doc/documentation-main.html#running-testng

Reply:
@nik-suri - yes thats what @juherr  was suggesting.

Reply:
@krmahadevan thanks.

It sounds like this option would then also require the separate support of 3rd party tools (such as the maven and gradle testng plugins), just like updating the schema would.
If both options are the same in this regard I would prefer updating the schema.

Please let me know your thoughts on this @juherr @krmahadevan

Reply:
@nik-suri - Not true. Adding a configuration parameter doesnt need anything extra to be done outside of TestNG. Build tools already have the mechanism to add configuration parameters to TestNG and so there's no work to be done from their side. Updating the schema part is not the same.

So lets stick to adding a configuration parameter instead of the schema modification.

Reply:
Upgrading the schema without its version is too risky. And having a new version to maintain will be too much extra work.
Please, fallback to the configuration way for the moment.

Reply:
@juherr @krmahadevan 

Ok, thank you for clarifying. I'll update the PR to use a configuration parameter instead.